README.md

```diff
@@ -1418,6 +1401,40 @@ domain with a *.* like this instead:
 set :sessions, :domain => '.foo.com'
 ```
 
+#### Choosing Your Own Session Middleware
+
+Note that `enable :sessions` actually stores all data in a cookie. This
+might not always be what you want (storing lots of data will increase your
+traffic, for instance). You can use any Rack session middleware: in order to
+do so, one of the following methods can be used:
+
+```ruby
+enable :sessions
+set :session_store, Rack::Session::Pool
+```
+
+Or to enable sessions with a hash of options:
+
+```ruby
+set :sessions, :expire_after => 2592000
+set :session_store, Rack::Session::Pool
+```
+
+Another option is to **not** call `enable :sessions`, but instead pull in your
+middleware of choice as you would any other middleware:
+
+```ruby
+use Rack::Session::Pool, :expire_after => 2592000
+```
+
+It is important to note that when using this method, session based protection (see 'Configuring attack protection') will not be enabled by default. The Rack middleware to do that will also need to be added:
```

Comment:
The reason that doesn't work is that `set :protection, :session => true` loads Rack::Protection::RemoteToken and Rack::Protection::SessionHijacking earlier in the middleware chain than Rack::Session::Pool. A session won't be available when RemoteToken checks so a RuntimeError is raised when a request is made.

I haven't been able to think of a reason calling `set :protection, :session => true` would be useful?